src/ui/EditTask.svelte

```diff
@@ -427,14 +427,14 @@ Availability of access keys:
     <!--  Description  -->
     <!-- --------------------------------------------------------------------------- -->
     <section class="tasks-modal-description-section">
-        <label for="description">Descrip<span class="accesskey">t</span>ion</label>
         <!-- svelte-ignore a11y-accesskey -->
         <textarea
             bind:value={editableTask.description}
             bind:this={descriptionInput}
             id="description"
             class="tasks-modal-description"
-            placeholder="Take out the trash"
+            placeholder="Description - take out the trash"
+            aria-label="Description"
```

Comment:
Why the addition of aria-label?

Reply:
Every `<textarea>`, `<input>` and `<select>` HTML element needs either
- a `<label>` associated with a `for` attribute to it
- an `aria-label` attribute

The `<label>` is gone now so I added this attribute for accessibility reasons.

Reply:
Is the label shown somehow - could it state the access key?